src/applications/vaos/referral-appointments/components/DateAndTimeContent.unit.spec.jsx

```diff
@@ -7,7 +7,7 @@ import { createReferralById, getReferralSlotKey } from '../utils/referrals';
 import { createDraftAppointmentInfo } from '../utils/provider';
 import { renderWithStoreAndRouter } from '../../tests/mocks/setup';
 
-describe('VAOS Component: DateAndTimeContent', () => {
+describe.skip('VAOS Component: DateAndTimeContent', () => {
```

Comment:
Should we ask the CIE team to look at these instead of just skipping them?

Reply:
Yes.